.github/workflows/continuous-integration.yml

```diff
@@ -58,6 +58,9 @@ jobs:
         with:
           node-version: 16
 
+      - name: Setup Postgres
+        uses: ./.github/actions/setup-postgres
+
```

Comment:
We don't need to run postgres to run the validation, this will only check prettier/eslint/tsc. 

```suggestion
```

Reply:
removed this from validation